src/daemon.rs

```diff
@@ -104,7 +104,12 @@ pub fn main_daemon(config: ConfigFile) -> anyhow::Result<()> {
             anon_identities: Arc::new(RwLock::new(AnonIdentities::new())),
             socket_recv_queues: Arc::new(DashMap::new()),
             unhandled_incoming: Arc::new(ConcurrentQueue::unbounded()),
-            forward_table: Arc::new(ForwardTable::new()),
+            forward_dests: Arc::new(
```

Comment:
you might want to rename this to something that makes it clear this is rendezvous related, like `registered_havens`